ramiro_serra/notes.md

```diff
@@ -0,0 +1,113 @@
+# Trainee program Week#2 notes
+
+After running the docker container, building with catkin, and launching `roslaunch ca_gazebo create_empty_house.launch`, we see gazebo launching and spawning a robot.
```

Comment:
The launch file doesn't exist, please check [here](https://github.com/RoboticaUtnFrba/create_autonomy/tree/kinetic-devel/ca_gazebo/launch).